src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -90,7 +90,7 @@ public enum Notification {
     /** Companions and observers of this user. */
     public List<RelatedUser> relatedUsers = new ArrayList<>();
 
-    /** Users that are dependent on this user. */
+    /** A list of users (their ids only) that are dependent on this user. */
     public List<String> dependents = new ArrayList<>();
```

Comment:
From @josh-willis-arcadis: "Another suggestion is to query a list of userIDs. that way if there are multiple dependents the frontend won't have to make separate calls"

Reply:
Updated: https://github.com/ibi-group/otp-middleware/pull/266/commits/deb32264726a15f90fe4a569c41c43f1113d74c5